### PR TITLE
test_lib_fork.py error

### PR DESCRIPTION
**Pull request by j00961415**

### CODE REVIEW

File: unittests/test_lib_fork.py [hunk 1]
```diff
@@ -109,11 +109,11 @@ def test_fetch_ready_fork_result_list(self):
...
         svl = 10
-        for key in range(1, 5):
-            pipe_ids[key] = fork_in_bg(sleep_fn, svl)
-            expected[key] = str(svl)
+        for i in range(1, 5):
+            pipe_ids[i] = fork_in_bg(sleep_fn, svl)
+            expected[i] = str(svl)
 
-        result = fetch_ready_fork_result_list(pipe_ids)
+        result = fetch_ready_fork_result_list({})
```
Comment (mambelli) [inline unittests/test_lib_fork.py:116]: Why the pipe_ids are removed here?

Comment (j00961415) [reply unittests/test_lib_fork.py:116]: pipe_ids = {}

Comment (j00961415) [reply unittests/test_lib_fork.py:116]: expected = {}

Comment (j00961415) [reply unittests/test_lib_fork.py:116]: if u just assigned it to pipe_ids will give you error
i have tried couple times and giving error
once i assigned it to {} it ran

**Comment by mambelli** [comment]
Better fix in #193

**Pull request closed without merging**